### PR TITLE
[CS] Code Style fixes for libraries Object operator not indented correctly

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue Object operator not indented correctly.

### Summary of Changes
- autofixed cases of Object operator not indented correctly

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

File: libraries/src/Updater/UpdateAdapter.php [hunk 1]
```diff
@@ -183,9 +183,9 @@ protected function getUpdateSiteName($updateSiteId)
 
 		$db = $this->parent->getDbo();
 		$query = $db->getQuery(true)
-					->select($db->qn('name'))
-					->from($db->qn('#__update_sites'))
-					->where($db->qn('update_site_id') . ' = ' . $db->q($updateSiteId));
+			->select($db->qn('name'))
+			->from($db->qn('#__update_sites'))
+			->where($db->qn('update_site_id') . ' = ' . $db->q($updateSiteId));
```
Comment (Quy) [inline libraries/src/Updater/UpdateAdapter.php:188]: Insert blank line below.

File: libraries/src/Installer/Adapter/ComponentAdapter.php [hunk 2]
```diff
@@ -803,8 +803,8 @@ public function uninstall($id)
 
 		// Remove the schema version
 		$query = $db->getQuery(true)
-					->delete('#__schemas')
-					->where('extension_id = ' . $id);
+			->delete('#__schemas')
+			->where('extension_id = ' . $id);
```
Comment (Quy) [inline libraries/src/Installer/Adapter/ComponentAdapter.php:807]: Insert blank line below.

File: libraries/src/Installer/Adapter/ComponentAdapter.php [hunk 3]
```diff
@@ -1176,10 +1176,10 @@ protected function _updateMenus($component_id, $clientId = null)
...
 		// to use the new component id.
 		$query = $db->getQuery(true)
-					->update('#__menu')
-					->set('component_id = ' . $db->quote($component_id))
-					->where('type = ' . $db->quote('component'))
-					->where('(' .
+			->update('#__menu')
+			->set('component_id = ' . $db->quote($component_id))
+			->where('type = ' . $db->quote('component'))
+			->where('(' .
```
Comment (Quy) [inline libraries/src/Installer/Adapter/ComponentAdapter.php]: Fix indentation below.

File: libraries/src/Updater/UpdateAdapter.php (hunk 1 above)
Comment (photodude) [reply libraries/src/Updater/UpdateAdapter.php:188]: You want a line between `$query = $db->` and `$db->setQuery($query);` ?

File: libraries/src/Installer/Adapter/ComponentAdapter.php (hunk 2 above)
Comment (photodude) [reply libraries/src/Installer/Adapter/ComponentAdapter.php:807]: you want a line between `$query = $db->` and `$db->setQuery($query);`?

File: libraries/src/Installer/Adapter/ComponentAdapter.php (hunk 3 above)
Comment (photodude) [reply libraries/src/Installer/Adapter/ComponentAdapter.php]: fixed, it's obvious with concatenation on the left-hand side.

File: libraries/src/Installer/Adapter/ComponentAdapter.php (hunk 2 above)
Comment (Quy) [reply libraries/src/Installer/Adapter/ComponentAdapter.php:807]: Yes, elsewhere it it like that.

File: libraries/src/Installer/Adapter/ComponentAdapter.php [hunk 4]
```diff
@@ -1176,13 +1176,13 @@ protected function _updateMenus($component_id, $clientId = null)
...
-					->where('type = ' . $db->quote('component'))
-					->where('(' .
-						'link LIKE ' . $db->quote('index.php?option=' . $option) . ' OR ' .
-						'link LIKE ' . $db->q($db->escape('index.php?option=' . $option . '&') . '%', false) .
-					')');
+			->update('#__menu')
+			->set('component_id = ' . $db->quote($component_id))
+			->where('type = ' . $db->quote('component'))
+			->where('('
+					. 'link LIKE ' . $db->quote('index.php?option=' . $option) . ' OR '
```
Comment (Quy) [inline libraries/src/Installer/Adapter/ComponentAdapter.php]: Reduce indentation.

File: libraries/src/Installer/Adapter/ComponentAdapter.php (hunk 2 above)
Comment (photodude) [reply libraries/src/Installer/Adapter/ComponentAdapter.php:807]: There is nothing in the code style that requires an additional line here. There is also nothing in the code style that forbids having the extra line. I wrote the example in the code style PHP SQL section and the only reason there are extra lines in the example is due to the comments. 

There are places throughout the repo (even throughout this file) showing with and without the extra line. Unless we include a requirement on this in the code style, I'm of the opinion that either way should be acceptable.

File: libraries/src/Installer/Adapter/ComponentAdapter.php (hunk 4 above)
Comment (photodude) [reply libraries/src/Installer/Adapter/ComponentAdapter.php]: reduced by one tab

File: libraries/src/Installer/Adapter/ComponentAdapter.php (hunk 2 above)
Comment (Quy) [reply libraries/src/Installer/Adapter/ComponentAdapter.php:807]: I don't know for sure. I am just going by the examples here:
https://docs.joomla.org/Selecting_data_using_JDatabase

Comment (photodude) [reply libraries/src/Installer/Adapter/ComponentAdapter.php:807]: So in that example it's the comments that require the extra line. There is one in that group that breaks that pattern but in my opinion there is no hard or fast rule beyond the new line before a comment .

**Pull request merged**